runtime/src/snapshot_utils/archive_format.rs

```diff
@@ -3,20 +3,17 @@ use {
     strum::Display,
 };
 
-pub const SUPPORTED_ARCHIVE_COMPRESSION: &[&str] = &["bz2", "gzip", "zstd", "lz4", "tar", "none"];
+// Publicly support "zstd" and "lz4", but retain support for "tar" for unit tests
+pub const SUPPORTED_ARCHIVE_COMPRESSION: &[&str] = &["zstd", "lz4"];
```

Comment:
I'd like a second opinion here - does it seem reasonable / not too confusing to do this. Namely,
- Enforce `zstd` or `lz4` (no `tar`) at the CLI level by passing `SUPPORTED_ARCHIVE_COMPRESSION` in for CLAP's `.possible_values()`
- Leave `ArchiveFormat::Tar` as an option so we can use it for unit tests

Reply:
> I'd like a second opinion here - does it seem reasonable / not too confusing to do this. Namely,
> - Enforce `zstd` or `lz4` (no `tar`) at the CLI level by passing `SUPPORTED_ARCHIVE_COMPRESSION` in for CLAP's `.possible_values()`
> - Leave `ArchiveFormat::Tar` as an option so we can use it for unit tests